specs/latest/1.0/index.html

```diff
@@ -1979,6 +1980,23 @@ <h4>Attributes</h4>
             The actual height of the drawing buffer. May be different from the
             <code>height</code> attribute of the <code>HTMLCanvasElement</code> if
             the implementation is unable to satisfy the requested width or height.
+        <dt>
+            <code class=attribute-name>
+                <a id="DOM-WebGLRenderingContext-colorSpace">
+                    colorSpace
+                </a>
+            </code>
+            of type <code><a href="https://html.spec.whatwg.org/multipage/canvas.html#predefinedcolorspace">PredefinedColorSpace</a></code>
+            <a href="#refsPREDEFINEDCOLORSPACE">(specification)</a>
+        <dd>
+            The color space in which the drawing buffer's contents are assumed to
+            have been produced. Changing this attribute affects the on-screen display
+            of the rendering context, but does not change the drawing buffer's
+            contents. This attribute also affects how <code>TexImageSource</code>
```

Comment:
Several of the optimized backends require that the color space for the buffer be "latched" when the buffer is allocated (e.g, CAMetalLayer does this). Allowing the color space to be changed after the buffer was allocated ends up significantly complicating the implementation (we have to somehow override the color space, or do a blit to a buffer of the right color space at the end of the frame).

Reply:
OK, let's change this to conform better to the per-platform implementations. How about:

`Changing this attribute causes the drawing buffer to be reallocated; its current contents are lost.`

?

Reply:
That sgtm.

Would we want to also want to separate this into drawingBufferColorSpace and unpackColorSpace, so that the developer can change the texture upload behavior without changing the drawing buffer behavior?

Reply:
Sure, it makes sense to retain flexibility in this API. I'll push an update to this PR making those adjustments.

Reply:
Current version of this PR contains the drawingBufferColorSpace / unpackColorSpace split.